content/en/security/application_security/setup/go/sdk.md

```diff
@@ -0,0 +1,205 @@
+---
+title: App and API Protection SDK for Go
+further_reading:
+- link: "https://pkg.go.dev/github.com/DataDog/dd-trace-go/v2/appsec"
+  tag: "Documentation"
+  text: "Go Security API docs"
+- link: "https://github.com/DataDog/appsec-go-test-app"
+  tag: "Source Code"
+  text: "Sample application used for exhaustive SDK testing"
+- link: "/security/application_security/add-user-info/"
+  tag: "Documentation"
+  text: "Adding user information to traces"
+- link: "/security/application_security/troubleshooting"
+  tag: "Documentation"
+  text: "Troubleshooting App and API Protection"
+---
+
+Sometimes, instrumenting your application automatically using [orchestrion][12] is not enough.
+That is why Datadog offer an API under [github.com/DataDog/dd-trace-go/v2/appsec][2] for more flexibility and more features
+that would not always be available.
+
+## Preamble: errors
+
+The API is designed to return errors when something goes wrong, so you can handle them in your code. More specifically
+an error of type [`events.BlockingSecurityEvent`][3] is returned when a blocking event occurs, such as a user, IP, or a request being blocked.
+
+**It is imperative to not write any response to the client when you receive a `BlockingSecurityEvent` error**, otherwise you are
+potentially sending sensitive information to an attacker. A helper called [`events.IsSecurityError`][4] is here to help
+this process. App and API Protection SDK is designed to send a blocking response in your stead (by default, a HTTP 403) when a
+`BlockingSecurityEvent` is returned. Customizing this behavior is possible, more available in
+the [Customize response to blocked requests][5] section.
+
+## HTTP body monitoring
+
+By default, a lot of Go HTTP Frameworks are not high level enough to provide parsed HTTP request & response bodies.
+This is why App and API Protection SDK provides a way to monitor the HTTP request and response bodies manually:
+
+- [`appsec.MonitorParsedHTTPBody`][8] to monitor the parsed request body.
+- [`appsec.MonitorHTTPResponseBody`][9] to monitor the response body.
+
+These functions automatically block the current request if asked from Datadog's UI, and return a
```

Comment:
```suggestion
These functions automatically block the current request if App and API Protection is enabled in the Datadog UI, and return a
```

Reply:
I wasn't sure what you meant by `if asked from Datadog's UI` but I believe calls to this function are ignored if
AAP is disabled or the given context is incorrect, so I suggested `if App and API Protection is enabled in the Datadog UI`.